plasmapy/particles/_factory.py

```diff
@@ -99,18 +99,27 @@ def _physical_particle_factory(
         raise TypeError("Particle information has not been provided.")
 
     for particle_type in (Particle, CustomParticle, ParticleList):
-        with contextlib.suppress(TypeError, InvalidParticleError):
+        with contextlib.suppress(TypeError, InvalidParticleError, ValueError):
+
+            # Do not interpret a string as the first positional argument
+            # as the symbol for a CustomParticle or something to be
+            # iterated over for a ParticleList.
+            if particle_type is not Particle and args and isinstance(args[0], str):
+                continue
+
             return particle_type(*args, **kwargs)
```

Comment:
Might be myself being tired but this whole block seems difficult to reason about!  This needs a refactor:
- let's enter `suppress` context once instead of within the for loop
- the case for `Particle` should probably be separate

Reply:
Yeah, this block is a bit hard to wrap one's mind around.  

The reason why `contextlib.suppress` is used within the `for` block is that it tries to create a `Particle`, `CustomParticle`, and `ParticleList` sequentially.  There's a chance for one of these exceptions to be raised each time.  If the `suppress` context were outside the `for` loop, then it would exit the context environment upon the first exception...I think.  If it failed to create a `Particle`, then it wouldn't try to create a `CustomParticle` or `ParticleList` afterwards.

I'll keep thinking about how to clean this up and reduce the cognitive complexity.